src/cli.ts

```diff
@@ -101,6 +107,7 @@ async function main({
     server,
     publish,
     'publish-tag': tag,
+    'allow-extra-props': allowExtraProps,
```

Comment:
Not really needed, since `yargs` will generate both `allow-extra-props: true` and `allowExtraProps: true`.

Reply:
That's good to know